go/exocom/test_helpers/index.go

```diff
@@ -0,0 +1,48 @@
+package testHelpers
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+
+	"github.com/Originate/exocom/go/utils"
+)
+
+// WaitForText reads from the stream waiting for the given text
+func WaitForText(stream io.ReadCloser, text string) error {
```

Comment:
What do you think about extracting this into a dedicated Go library? We will need this on a ton of projects including Exosphere, Morula, etc.